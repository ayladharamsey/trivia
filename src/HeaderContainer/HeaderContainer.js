import React from 'react';
import './HeaderContainer.css';
import Nav from '../Nav/Nav';
import UserProfile from '../UserProfile/UserProfile'

const HeaderContainer = ({ name, quote, rating, signOut}) => {
    return (
        <section>
            <Nav signOut={ signOut }/>
            <UserProfile name={ name } quote={ quote } rating={ rating }/>
        </section>
    )
}

export default HeaderContainer;