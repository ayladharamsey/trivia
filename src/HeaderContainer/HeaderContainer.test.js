import React from 'react';
import { shallow } from 'enzyme';
import HeaderContainer from './HeaderContainer';

describe('HeaderContainer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<HeaderContainer 
      name = 'Ayla'
      quote =  'she is an an average human'
      rating = 'master'
      signOut = {jest.fn()}
    />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });


});