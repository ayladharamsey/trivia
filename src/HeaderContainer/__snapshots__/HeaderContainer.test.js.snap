// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderContainer should match the snapshot 1`] = `
<section
  className="header-component"
>
  <Nav
    signOut={[MockFunction]}
  />
  <UserProfile
    name="Ayla"
    quote="she is an an average human"
    rating="master"
  />
</section>
`;
